tools/cmake/toolchains/ARM.cmake

```diff
@@ -55,6 +55,8 @@ function(mbed_set_toolchain_options target)
 
     list(APPEND link_options
         "${MBED_STUDIO_ARM_COMPILER}"
+        # Add linker option to generate a map file.
```

Comment:
Please remove that comment. We do not have comments for the other flags and I do not see why this is special.